types/nova-editor-node/index.d.ts

```diff
@@ -5,6 +5,7 @@
 /// <reference no-default-lib="true"/>
 /// <reference lib="es2020" />
 /// <reference lib="WebWorker" />
+/// <reference types="vscode-languageserver-types" />
```

Comment:
How is this working? Doesn't this package export its types, not globally declare them?

Reply:
Oh, good catch; I just forgot to remove this. It does indeed, and the relevant line is the one further down the file that does it with an `= import('vscode-languageserver-types').TheType` instead.

```suggestion
```